src/main/java/com/kammatysiak/medicalclinic/model/entity/Patient.java

```diff
@@ -35,6 +38,9 @@ public class Patient {
     private LocalDate birthday;
     @Column(name = "MODIFY_DATE")
     private LocalDateTime modifyDate;
+    @OneToMany
+    @JoinColumn(name = "VISITS")
```

Comment:
Wydaje mi sie ze jest to tutaj niepotrzebne.
@OneToMany(mappedBy = "patient") <- patient bo tak nazywa sie pole w Visit